src/java/apiview-java-processor/src/main/java/com/azure/tools/apiview/processor/model/APIListing.java

```diff
@@ -18,14 +18,19 @@
     @JsonProperty("Name")
     private String name;
 
-//    @JsonProperty("Language")
-    @JsonIgnore
+    @JsonProperty("Language")
     private String language;
 
+    @JsonProperty("PackageName")
+    private String packageName;
+
+    @JsonProperty("PackageVersion")
```

Comment:
You can remove this property for version for the time being. I haven't added it yet on APIView tool.

Reply:
Once I do that it should be OK to merge this pr?